golang/dep
# ADDED example comment to toml file on init #343

This is in reference to #343 

# To test

1. Create a blank project using `$ dep init` ensuring no dependencies are detected within the code. I.e a blank directory. 
2. Check `Gopkg.toml` for some example text, commented out.
